.github/workflows/generate-docs.yml

```diff
@@ -38,6 +38,9 @@ jobs:
                 target: dev
                 EOF
             - run: dbt deps
+            - run: dbt seed
+            - run: dbt run
+            - run: dbt test
```

Comment:
for the future - we could leverage this to put some sort of badge showing all the tests are passing :)